javascript/javascript1/week1/main.css

```diff
@@ -0,0 +1,337 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+body {
+    display: flex;
```

Comment:
It would be a better solution to wrap the whole content and style it using flexbox

Reply:
Yes true .